README.md

```diff
@@ -2,6 +2,7 @@
 [![Build Status](https://travis-ci.org/Statoil/camille.svg?branch=master)](https://travis-ci.org/Statoil/camille)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/61db6e0137e743db84041b1239436c39)](https://www.codacy.com/app/equinor_sib/camille?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Statoil/camille&amp;utm_campaign=Badge_Grade)
 [![Codacy Badge](https://api.codacy.com/project/badge/Coverage/61db6e0137e743db84041b1239436c39)](https://www.codacy.com/app/equinor_sib/camille?utm_source=github.com&utm_medium=referral&utm_content=Statoil/camille&utm_campaign=Badge_Coverage)
+[![MutMut Badge](https://img.shields.io/badge/dynamic/json.svg?label=Surviving%20mutants&url=https%3A%2F%2Fs3-eu-west-1.amazonaws.com%2Fequinor-sib%2Fcamille%2Fmutmut_report.json&query=%24..survival_rate&colorB=blue)](https://mutmut.readthedocs.io/en/latest/)
```

Comment:
Maybe you can define color in separate bucket, if you want? :) (And if it works that way)

Reply:
Good idea. I will look into it

Reply:
I got my hopes up early on, but it seems the colour setting on the badge isn't dynamic.